spec/api.yaml

```diff
@@ -112,25 +112,28 @@ paths:
                 v:
                   type: string
                   description: A base64URL-encoded bencoded DNS packet containing the DID Document.
-                retention_proof:
+                retention_solution:
                   type: string
-                  description: A retention proof calculated according to the spec-defined retention proof algorithm.
+                  description: A retention solution calculated according to the spec-defined retention solution algorithm.
               required: [ did, sig, seq, v ]
       responses:
         "202":
           description: Accepted. The server has accepted the request as valid and will publish it to the
           content:
             application/json:
               schema:
-                type: string
+                type: object
+                properties:
+                  expiry:
+                    type: number
         "400":
           description: Invalid request.
           content:
             application/json:
               schema:
                 type: string
         "401":
-          description: Invalid signature.
+          description: Invalid signature or retention solution.
```

Comment:
why wouldn't an invalid retention solution be a 400?

Reply:
agree, updating